sphinx/writers/latex.py

```diff
@@ -1797,9 +1787,15 @@ def visit_reference(self, node):
                 else:
                     self.context.append('}}}')
         else:
-            self.builder.warn('unusable reference target found: %s' % uri,
-                              (self.curfilestack[-1], node.line))
```

Comment:
This restriction began from https://github.com/sphinx-doc/sphinx/commit/c5580ebd9115dc8dd1e76a8ac25cea95bc3db2ce#diff-c524933be923016c87a9da437a0e7537R478 (v0.1.61611).
I think there are no reason to restrict hyperlinks by URI scheme.
In this commit, I drop the restriction.

Note: `URI_SCHEMES` is only used for show_url feature now. It might be better to replace it by regexp.